pkg/storage/tpr/install_types_test.go

```diff
@@ -177,7 +177,7 @@ func TestInstallTypesPolling(t *testing.T) {
 
 	for _, name := range getCallArgs {
 		if name == serviceBrokerTPR.Name || name == serviceInstanceTPR.Name {
-			t.Errorf("Failed to skip polling for resource that failed to install")
+			t.Error("Failed to skip polling for resource that failed to install", name)
```

Comment:
missing a %q? and should call Errorf()? Even if Error() concats them it would be nice to use Errorf()

Reply:
What is with your quote obsession?

Reply:
LOL  once too many times I even misled by an error message where a name looked like an english word

Reply:
this one still needs to be addressed